### PR TITLE
Added program to show ASCII value of character.

### PR DESCRIPTION
**Pull request by karan13421**
i am new to open source if any mistake please give me feedback

### CODE REVIEW

**Comment by rishiip** [comment]
Thanks @karan13421 for showing your interest in @Novice-Paradigm! :bowtie:
   
Feel free to join us over [Slack](https://join.slack.com/t/noviceparadigm/shared_invite/enQtNDIyMjczMzA1MzgyLWNlYmY2NjE2MGM3OTZkY2ZkNWI0ZDY2MjE0YTk2NDhjNzRmOTMyYjM0YjYwYmMxN2QxZjRiMTYzNmMyM2JlNTk) or on [Facebook](https://www.facebook.com/noviceparadigm/) for communicate with other contributors. :v:

   Please apply for [Hacktoberfest](https://hacktoberfest.digitalocean.com/). It's a fest celebrating open source where you get to win a free hacktoberfest t-shirt! :tshirt: Know more about it at the link itself.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Char_to_ASCII.out
```
Comment (rishiip) [inline .gitignore]: Please remove this modifications.

File: Char_to_ASCII/Char_to_ASCII.cpp [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+int main()
+{
+    char abc;
+    cout<<"Enter the character ";
+    cin>>abc;
+    cout<<"ASCII of character "<<abc<<" = "<<(int)abc;
+    return 0;
+}
+
+
```
Comment (rishiip) [inline Char_to_ASCII/Char_to_ASCII.cpp]: Remove this blank lines.

File: Char_to_ASCII/Char_to_ASCII.cpp [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+
+using namespace std;
+
+int main()
+{
+    char abc;
```
Comment (rishiip) [inline Char_to_ASCII/Char_to_ASCII.cpp]: Give readable variable name.

**Comment by karan13421** [comment]
Resolved requested issues and also added two programs:
Program to check whether the number is prime or not.
Program to find the prime numbers between the range

**Pull request merged**